lib/generators/administrate/install/install_generator.rb

```diff
@@ -9,7 +9,7 @@ class InstallGenerator < Rails::Generators::Base
       def create_dashboard_controller
         copy_file(
           "application_controller.rb",
-          "app/controllers/admin/application_controller.rb"
+          "app/controllers/#{Administrate::Config.admin_directory}/application_controller.rb"
```

Comment:
Line is too long. [93/80]<br>Put a comma after the last parameter of a multiline method call.